repldex/config/__init__.py

```diff
@@ -44,4 +44,7 @@
 	BLACKLIST_IDS = [int(item.split()[0]) for item in f.read().splitlines()]
 	BLACKLISTED_IDS = BLACKLIST_IDS
 
+with open(config_path + 'featured.txt') as f:
+	FEATURED = [int(item.split()[0]) for item in f.read().splitlines()]
```

Comment:
Entry id's are strings (uuid's to be exact) not ints, int() would breakity break break it